docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -402,6 +402,10 @@ Note that the diagnostics bundle is intended for debugging purposes only, its st
 IMPORTANT: {agent} attempts to automatically redact credentials and API keys when creating diagnostics. 
 Please review the contents of the archive before sharing to ensure that there are no credentials in plain text.
 
+IMPORTANT: The zip archive containing diagnostics information will contain the raw events of documents sent to the output. By default, it will log only the failing events as `warn`.
+When enabling `debug` logging level, it will log all the events. 
+Please review the contents of the archive before sharing to ensure that there is no sensible information in them.
```

Comment:
```suggestion
IMPORTANT: The ZIP archive containing diagnostics information will include the raw events of documents sent to the {agent} output.
By default, it will log only the failing events as `warn`.
When the `debug` logging level is enabled, all events are logged.
Please review the contents of the archive before sharing to ensure that no sensitive information is included.
```

Looks good Luca! I'd suggest just a very small rewording. Here's how it would look:

![Screenshot 2024-08-13 at 11 51 44 AM](https://github.com/user-attachments/assets/239e37ad-e7ec-438a-b9e0-0f2640fa0055)